### ISSUE TITLE
Create broader colour palette in the EDS Power BI template

### ISSUE THREAD

**Issue by BeckyBrekke** [equinor/design-system#2475]
Email received from Christophe Roth Maingourd 07.09.22: 
![image.png](https://images.zenhubusercontent.com/6075453285eb60bb55b90568/67631813-fcc1-4896-981d-5222889e95f8)

**Comment by BeckyBrekke** [comment]
I responded that I could make a feature request on his behalf, and that we have an issue to revisit data visualization colors, but has not been prioritized yet. https://app.zenhub.com/workspaces/equinor-design-system-604f57c09e350a0011b654f4/issues/equinor/design-system/2254

**Comment by benteljones** [comment]
Rebecca has collected a lot of good background material. I have started with the work of building all graphic variants (columns, curves, pies etc.) to test out new colour palettes. What works on one does not work on others. The thin curve diagrams in particular create challenges. Alarm colours should be avoided, as this limits the colour palette a lot. In addition, the colours should be perceived as equal value (if you don't want to highlight one/several). This is particularly challenging, because it limits the use of light/darkness etc.. But we will find a way!![Skjermbilde 2022-10-25 kl. 11.05.10.png](https://images.zenhubusercontent.com/617137275a2f3b5f9e07865a/b52ac606-47e5-49a3-ac15-9a34d5281de9)

**Comment by benteljones** [comment]
https://statoilsrm.sharepoint.com/sites/Designsystem/_layouts/15/Doc.aspx?sourcedoc={4777fb48-6bb9-4fce-87a7-8a526aa7057c}&action=edit&wd=target%28Pre%20work%20-%20examples%20in%20Equinor.one%7Cb76bc972-d15c-433a-9464-609ffe53a029%2FDashboards%7Cd05896b6-80e9-4222-9a02-0d410dd40716%2F%29&wdorigin=NavigationUrl

**Comment by BeckyBrekke** [comment]
Another request from a Power BI user came in last week. I will let him know we have some work in progress: From: Yousef Shahin <YOSH@equinor.com> 
Sent: Friday, November 4, 2022 3:30 PM
To: Rebecca Sue Brekke <rbre@equinor.com>
Cc: Timothy DeJesu <tdej@equinor.com>
Subject: Equinor Power BI Template

Hi Rebecca, 

I hope this message finds you well, and great job setting up the EDS Power BI Template. It is a great introductory tool and super easy to follow. 


I am reaching out in the hopes that you can suggest what I should do to expand the number of 'Equinor Colors' to select from within power bi. When applying the Equinor Theme to some charts with a large number data series, it will take this variation of colors:
![image.png](https://images.zenhubusercontent.com/6075453285eb60bb55b90568/4eb89366-4f16-4483-959c-4ae26858d3ec)
 


And iterate through all the data series to create a chart that looks like this:
 ![image.png](https://images.zenhubusercontent.com/6075453285eb60bb55b90568/e2904620-f572-4d58-be7d-273f61ecb4d3)

As you can see the colors are a bit redundant, so I am wondering if there is any way around this so that we can still use the Equinor Color theme but have a wider variety of colors that are more easily distinguishable. 


Thanks!


•	Yousef Shahin

**Comment by BeckyBrekke** [comment]
@benteljones 
There is a new colleague that I have had some meetings with that has been hired to work on analytics and data visualization best practices, guidelines, learning. We discussed ongoing work here in EDS, and she recommended some material that could be useful. Will paste it in here: 
From Laura Garrison: 
This writer is fantastic, and here are a few useful links that the designer you mentioned may find useful

https://blog.datawrapper.de/colors-for-data-vis-style-guides/#include-brand-colors (this is the link I mentioned on our call, and this entire article is really wonderful). 

https://blog.datawrapper.de/10-ways-to-use-fewer-colors-in-your-data-visualizations/

What I would love to do in my role here is to condense a lot of the points she makes across these blogs that are really Equinor-specific, and then as "further reading" link out to the source links (so she gets the credit she deserves, as well as for people here who are just really interested in the topic).

**Comment by BeckyBrekke** [comment]
Another comment - there is something called [WebViz in Equinor](https://miro.com/app/board/uXjVMWuJYA8=/?moveToWidget=3458764550664881521&cot=14) that I haven't hear that much from lately, but a few years ago they were looking for more distinct colors to use and would go away from brand. They decided to use to the d3.schemeCategory10 ones that are here: 
https://github.com/d3/d3-scale-chromatic#schemeCategory10
![Image](https://user-images.githubusercontent.com/34247620/229777453-7e685a5f-7adc-4040-a6b9-ed6f497b9f82.png)

**Comment by benteljones** [comment]
Great if we can test a proposal in P BI together, @trondeide 😊

**Comment by benteljones** [comment]
The program's limitations create major challenges in meeting the various needs of the users. But we will create a simplified solution at this stage and then we have to follow this up with more tasks, guidelines and examples little by little.

**Comment by benteljones** [comment]
Before we can do a proper review, @trondeide  and I have to learn about the program's limitations. We also open a new task;  guidelines how the colours should be used. This is a big task, because it depends on many different contexts.
